### ISSUE TITLE
Dirty check in array-like codemods is incorrect

### ISSUE THREAD

**Issue by 43081j** [es-tooling/module-replacements-codemods#88]
In the `transformArrayMethod`, i think the dirty check is wrong:

https://github.com/es-tooling/module-replacements-codemods/blob/f5f2b4562561264cace7ebe28a90a11bb6e2b36b/codemods/shared.js#L308-L310

you can see we only consider the change to be `dirty` if we remove any call expressions

however, we may have removed imports but no call expressions. so i think it should be considered dirty if we remove imports

basically:

```ts
const { identifier, wasRemoved } = removeImport(method, root, j);

let dirtyFlag = wasRemoved;

// eixsting logic
```
